docs/modules/reference/pages/configuration/core-docker.adoc

```diff
@@ -4,93 +4,432 @@
 
-.Karaf Shell configuration in org.apache.karaf.shell.cfg
+=== Karaf Shell configuration
+
+These options are written to `$\{OPENNMS_HOME}/etc/org.apache.karaf.shell.cfg`.
+
 [options="header, %autowidth"]
 |===
-| Environment variable     | Description                        | Required | Default value
-| `OPENNMS_karaf_SSH_HOST` | Listen interface for Karaf shell | optional | `0.0.0.0`
-| `OPENNMS_karaf_SSH_PORT` | SSH port for Karaf shell       | optional | `8101`
+| Environment variable | Description |  Default value
+
+3+| *Optional*
+
+| OPENNMS_KARAF_SSH_HOST
+| Listen interface for Karaf shell
+| 0.0.0.0
+
+| OPENNMS_KARAF_SSH_PORT
+| SSH port for Karaf shell
+| 8101
 |===
 
-.Cassandra and Newts configuration in opennms.properties.d/_confd.newts.properties
+=== Cassandra and Newts configuration
+
+These options are written to `$\{OPENNMS_HOME}/etc/opennms.properties.d/_confd.newts.properties`.
+
 [options="header, %autowidth"]
 |===
-| Environment variable          | Description                                                                       | Required | Default value
-| `REPLICATION_FACTOR`          | Set Cassandra replication factor for the newts keyspace if Newts is used.      | optional | `1`
-| `OPENNMS_CASSANDRA_HOSTNAMES` | A comma-separated list with Cassandra hosts for Newts                         | optional | `localhost`
-| `OPENNMS_CASSANDRA_KEYSPACE`  | Name of the keyspace used by Newts                                              | optional | `newts`
-| `OPENNMS_CASSANDRA_PORT`      | Cassandra server port                                                           | optional | `9042`
-| `OPENNMS_CASSANDRA_USERNAME`  | Username with access to Cassandra                                              | optional | `cassandra`
-| `OPENNMS_CASSANDRA_PASSWORD`  | Password for user with access to Cassandra                                      | optional | `cassandra`
+| Environment variable | Description | Default value
+
+3+| *Optional*
+
+| REPLICATION_FACTOR
+| Set Cassandra replication factor for the newts keyspace if Newts is used.
+| 1
+
+| OPENNMS_CASSANDRA_HOSTNAMES
+| A comma-separated list with Cassandra hosts for Newts
+| localhost
+
+| OPENNMS_CASSANDRA_KEYSPACE
+| Name of the keyspace used by Newts
+| newts
+
+| OPENNMS_CASSANDRA_PORT
+| Cassandra server port
+| 9042
+
+| OPENNMS_CASSANDRA_USERNAME
+| Username with access to Cassandra
+| cassandra
+
+| OPENNMS_CASSANDRA_PASSWORD
+| Password for user with access to Cassandra
+| cassandra
 |===
 
-== Directory conventions
+=== Daemon control
+
+Select daemons are able to be enabled or disabled through the container definition with the following boolean variables.
 
 [options="header, %autowidth"]
 |===
-| Mountpoint             | Description
-| `/opt/opennms-overlay` | Allows you to overwrite files relative to `/opt/opennms`.
-| `/opennms-data`        | Directory with RRDTool/JRobin files and generated PDF reports sent to the file system.
+| Environment variable
```

Comment:
This looks odd as a table with only one cell. Maybe make it a bulleted list?

Reply:
That can be done.  I had made it a table to fit with the format of the other sections, but a list does make more sense here.